cub/warp/specializations/warp_scan_smem.cuh

```diff
@@ -70,7 +70,11 @@ struct WarpScanSmem
     };
 
     /// Storage cell type (workaround for SM1x compiler bugs with custom-ops like Max() on signed chars)
-    typedef typename If<((Equals<T, char>::VALUE || Equals<T, signed char>::VALUE) && (PTX_ARCH < 200)), int, T>::Type CellT;
+    using CellT =
```

Comment:
This should just be `using CellT = T` since `int` was used only for >sm_20. The comment should be updated, too.